packages/Parallel/WorkerRunner.php

```diff
@@ -129,23 +131,24 @@ public function run(Encoder $encoder, Decoder $decoder, Configuration $configura
 
     /**
      * @param array{system_errors: SystemError[], file_diffs: FileDiff[]}|mixed[] $errorAndFileDiffs
-     * @return array{system_errors: SystemError[], file_diffs: FileDiff[]}
      */
-    private function processFile(File $file, Configuration $configuration, array $errorAndFileDiffs): array
+    private function processFile(File $file, Configuration $configuration, array $errorAndFileDiffs): ProcessFileResult
```

Comment:
not sure this kind of change will get accepted, see https://github.com/rectorphp/rector/issues/7937